go.mod

```diff
@@ -62,5 +62,3 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
-
```

Comment:
This should be kept here, and we need to update the dependency under the path client/ as well.

This is a tricky one.

Reply:
Updated the dependencies for `./client` and `csi-lib-utils`.